giraffe/src/utils/PlotEnv.ts

```diff
@@ -130,10 +130,10 @@ export class PlotEnv {
   }
 
   public get yScale(): Scale<number, number> {
-    const getYScaleMemoized = this.fns.get('yScale', getLinearScale)
+    const scale = getScale(this.config.yScale)
```

Comment:
To memoize, it would be something like this:
<code>const getYScaleMemoized = this.fns.get(&#96;yScale_${this.config.yScale}&#96;, getScale(this.config.yScale))</code>